app/mailers/custom_devise_mailer.rb

```diff
@@ -7,4 +8,8 @@ def confirmation_instructions(record, token, options = {})
     @confirmation_period = user_decorator.confirmation_period
     super
   end
+
+  def attach_images
+    attachments.inline['logo.png'] = File.read('app/assets/images/logo.png')
+  end
```

Comment:
maybe we shouldn't do an image at all and just have it be 'LOGIN.GOV' -- the bulk of the image is text after all